### PR TITLE
feat(datepicker): Add directive for inline datepicker

### PR DESCRIPTION
**Pull request by martintro**
Closes valor-software/ngx-bootstrap#3955

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated tests.
 - [x] added/updated API documentation.
 - [x] added/updated demos.

### CODE REVIEW

**Review by alexborghesi — DISMISSED** [review]
looks good

**Comment by martintro** [comment]
Closes valor-software/ngx-bootstrap#3958

**Comment by amorenew** [comment]
@martintro I wonder if you could add small sample how could we use this inline datepicker?
is it will be `<datepicker></datepicker>` or how it will be?
Thanks

**Comment by martintro** [comment]
@amorenew Please have a look at the demo files in the PR for example.
It’s like ```<bs-datepicker-inline></bs-datepicker-inline>```

**Comment by amorenew** [comment]
@martintro Yeah I saw it nice
`<bs-datepicker-inline></bs-datepicker-inline>`
well done 👍 
I have a small suggestion I think it could be nicer to be `<bs-datepicker></bs-datepicker>`

**Comment by shyamal890** [comment]
Can another reviewer please review and merge this pull request to the main repo.

**Comment by valorkin** [comment]
@martintro LGTM!

**Comment by ishworthapaliya** [comment]
Any update on this one?

**Comment by martintro** [comment]
@ishworthapaliya I hope so soon. I'm waiting for approval :)

**Comment by x1unix** [comment]
@valorkin, please review this PR.

**Comment by pmramos** [comment]
Could really use this!

File: demo/src/app/components/+datepicker/demos/inline-datepicker/inline-datepicker.component.html
Comment (ludmilanesvitiy) [inline demo/src/app/components/+datepicker/demos/inline-datepicker/inline-datepicker.component.html]: Not need to add description to the DEMO inside the template for component, better will be move it as a description inside `datepicker-section.list.ts`

Comment (Domainv) [reply demo/src/app/components/+datepicker/demos/inline-datepicker/inline-datepicker.component.html]: fixed

**Review by Domainv — DISMISSED** [review]

**Review by ludmilanesvitiy — DISMISSED** [review]

**Comment by martintro** [comment]
When will this be merged?

File: src/index.ts
Comment (valorkin) [inline src/index.ts]: this type of import usually kills system js builds

**Review by Domainv — DISMISSED** [review]

File: src/index.ts
Comment (valorkin) [reply src/index.ts]: @Domainv this is why I didn't yet merged

**Review by valorkin — CHANGES_REQUESTED** [review]
please fix imports and ready for merge

File: src/index.ts
Comment (Domainv) [reply src/index.ts]: fixed

**Review by valorkin — APPROVED** [review]
Thanks!

**Pull request merged**
